test/envtest/setup.go

```diff
@@ -0,0 +1,85 @@
+package envtest
```

Comment:
Should we put the code for creating and configuring envtest env here or in `pkg/util/test`?

Reply:
Any reason to put this there? I believe that `test/envtest/setup.go` is a sound choice for this